app/models/admin_flash.rb

```diff
@@ -1,13 +1,15 @@
 class AdminFlash < ActiveRecord::Base
-  
+
   validates :message, presence: true
   validates :expires, presence: true
 
   def self.unexpired_messages
-    self.order("created_at desc").where("expires >= ?", Time.now)
+    self.order('created_at desc').where('expires >= ?', Time.now)
   end
 
-  def self.showable_messages disabled_flash_ids
-    unexpired_messages.select{ |msg| !disabled_flash_ids.include?(msg.id) }
+  def self.showable_messages(disabled_flash_ids)
+    unexpired_messages.select do |message|
```

Comment:
Can we use `reject` instead to get rid of `!` ?

IMO, it makes it a bit hard to understand the code